alot/commands/thread.py

```diff
@@ -1075,6 +1076,25 @@ def apply(self, ui):
             ui.apply_command(ChangeDisplaymodeCommand(visible='toggle'))
 
 
+@registerCommand(MODE, 'retagprompt')
+class RetagPromptCommand(Command):
+
+    """prompt to retag selected message\'s tags"""
```

Comment:
the escape is unnecessary, right?

Reply:
Yes, it's just following the existing style in `search.py`.
Thinking of it: AFAIK there is no way to select multiple threads as once, so the plural _threads'_ in `search.py` appears to be incorrect.